Update get-partitionsupportedsize.md (#1096)
MicrosoftDocs/windows-powershell-docs: docset/windows/storage/get-partitionsupportedsize.md

SizeMin                                                 SizeMax 
-----------                                             -----------
```

> This command will show the actual value to see the minimum and maximum size for the partition (SizeMin and SizeMax).

Reviewer: ```suggestion
This command will show the actual minimum and maximum size of the partition (SizeMin and SizeMax).
```
